### PR TITLE
Turkish fields refactored

### PR DESCRIPTION
**Pull request by senocak**

### CODE REVIEW

**Review by asolntsev — CHANGES_REQUESTED** [review]
@senocak All good, but why last name has disappeared?

I cannot find these last names anymore:
>       last_name: [Şen, Kandemir, Çevik, Erkuran, Tüten, Öztürk, Yüzbaşıoğlu, Vural, Yücel, Sönmez, Ertekin, Dede, Uyanık, Aslan, Akbulut, Orhon, Uz, Yavuz, Erdem, Kulaç, Kaya, Selvi, Akpınar, Abacıoğlu, Çay, Işık, Özer, Tahtacı, Büyükcam, Kulaksız, Aksel, Eroğlu, Karakum, Dal, Öcal, Ayhan, Yarbil, Canacankatan, Gümüşay, Murt, Halhallı, Uluöz, Şeyhanlı, Çalışkantürk, Yılmaz, Saraçoğlu, Sezer, Demir, Kayayurt, Sürüm, Yavaşi, Turgut, Tanrıkulu, Barbaros, Aldinç, Tekin, Gülşan, Küfeciler, Almacıoğlu, Çildir, Türkdoğan, Öner, Şeliman, Yaman, Atik, Giray, Yalçınkaya, Kılıç, Şentürk, Karabağ, Değirmenci, Boduroğlu, Yıldız, Güler, Eraslan, Üzer, Pişirgen, Adanır, Koç, Korkmaz, Yenidoğan, Aydoğan, Acarbulut, Erge, Erdoğan, Öğüt, Kuşku, Pektaş, Kayacan, Gülen, Akküçük, Candan, Temel, Yenigün, Yıldırım]

**Comment by senocak** [comment]
> @senocak All good, but why last name has disappeared?
> 
> I cannot find these last names anymore:
> 
> > ```
> >   last_name: [Şen, Kandemir, Çevik, Erkuran, Tüten, Öztürk, Yüzbaşıoğlu, Vural, Yücel, Sönmez, Ertekin, Dede, Uyanık, Aslan, Akbulut, Orhon, Uz, Yavuz, Erdem, Kulaç, Kaya, Selvi, Akpınar, Abacıoğlu, Çay, Işık, Özer, Tahtacı, Büyükcam, Kulaksız, Aksel, Eroğlu, Karakum, Dal, Öcal, Ayhan, Yarbil, Canacankatan, Gümüşay, Murt, Halhallı, Uluöz, Şeyhanlı, Çalışkantürk, Yılmaz, Saraçoğlu, Sezer, Demir, Kayayurt, Sürüm, Yavaşi, Turgut, Tanrıkulu, Barbaros, Aldinç, Tekin, Gülşan, Küfeciler, Almacıoğlu, Çildir, Türkdoğan, Öner, Şeliman, Yaman, Atik, Giray, Yalçınkaya, Kılıç, Şentürk, Karabağ, Değirmenci, Boduroğlu, Yıldız, Güler, Eraslan, Üzer, Pişirgen, Adanır, Koç, Korkmaz, Yenidoğan, Aydoğan, Acarbulut, Erge, Erdoğan, Öğüt, Kuşku, Pektaş, Kayacan, Gülen, Akküçük, Candan, Temel, Yenigün, Yıldırım]
> > ```

I've missed that 

> @senocak All good, but why last name has disappeared?
> 
> I cannot find these last names anymore:
> 
> > ```
> >   last_name: [Şen, Kandemir, Çevik, Erkuran, Tüten, Öztürk, Yüzbaşıoğlu, Vural, Yücel, Sönmez, Ertekin, Dede, Uyanık, Aslan, Akbulut, Orhon, Uz, Yavuz, Erdem, Kulaç, Kaya, Selvi, Akpınar, Abacıoğlu, Çay, Işık, Özer, Tahtacı, Büyükcam, Kulaksız, Aksel, Eroğlu, Karakum, Dal, Öcal, Ayhan, Yarbil, Canacankatan, Gümüşay, Murt, Halhallı, Uluöz, Şeyhanlı, Çalışkantürk, Yılmaz, Saraçoğlu, Sezer, Demir, Kayayurt, Sürüm, Yavaşi, Turgut, Tanrıkulu, Barbaros, Aldinç, Tekin, Gülşan, Küfeciler, Almacıoğlu, Çildir, Türkdoğan, Öner, Şeliman, Yaman, Atik, Giray, Yalçınkaya, Kılıç, Şentürk, Karabağ, Değirmenci, Boduroğlu, Yıldız, Güler, Eraslan, Üzer, Pişirgen, Adanır, Koç, Korkmaz, Yenidoğan, Aydoğan, Acarbulut, Erge, Erdoğan, Öğüt, Kuşku, Pektaş, Kayacan, Gülen, Akküçük, Candan, Temel, Yenigün, Yıldırım]
> > ```
Yes, you are right. I accidentally deleted it and put them back.

**Review by asolntsev — APPROVED** [review]

File: src/main/resources/tr.yml [hunk 1]
```diff
@@ -2,10 +2,14 @@
...
       last_name: [Şen, Kandemir, Çevik, Erkuran, Tüten, Öztürk, Yüzbaşıoğlu, Vural, Yücel, Sönmez, Ertekin, Dede, Uyanık, Aslan, Akbulut, Orhon, Uz, Yavuz, Erdem, Kulaç, Kaya, Selvi, Akpınar, Abacıoğlu, Çay, Işık, Özer, Tahtacı, Büyükcam, Kulaksız, Aksel, Eroğlu, Karakum, Dal, Öcal, Ayhan, Yarbil, Canacankatan, Gümüşay, Murt, Halhallı, Uluöz, Şeyhanlı, Çalışkantürk, Yılmaz, Saraçoğlu, Sezer, Demir, Kayayurt, Sürüm, Yavaşi, Turgut, Tanrıkulu, Barbaros, Aldinç, Tekin, Gülşan, Küfeciler, Almacıoğlu, Çildir, Türkdoğan, Öner, Şeliman, Yaman, Atik, Giray, Yalçınkaya, Kılıç, Şentürk, Karabağ, Değirmenci, Boduroğlu, Yıldız, Güler, Eraslan, Üzer, Pişirgen, Adanır, Koç, Korkmaz, Yenidoğan, Aydoğan, Acarbulut, Erge, Erdoğan, Öğüt, Kuşku, Pektaş, Kayacan, Gülen, Akküçük, Candan, Temel, Yenigün, Yıldırım]
-      prefix: [Sn., Av., Dr.]
+      prefix: ["Bay", "Bayan", "Hanım", "Bayan", "Doktor"]
+      suffix: []
```
[the remark this answers is not in the document]
Comment (kingthorin) [reply src/main/resources/tr.yml:9]: This seems to be the cause of the build failure:\08:18:53.700 [ERROR] Errors: 
```
08:18:53.701 [ERROR]   FakerIntegrationTest.testAllFakerMethodsThatReturnStrings:100->testAllMethodsThatReturnStringsActuallyReturnStrings:122 Runtime Test for method public java.lang.String net.datafaker.providers.base.Name.suffix() and object Name(Faker@4b511ea8)@12ace6f4 was failed for locale tr [thread: FakerIntegrationTest.testAllFakerMethodsThatReturnStrings#16902#]
08:18:53.701 [ERROR] Tests run: 19451, Failures: 0, Errors: 1, Skipped: 0
```

[the remark this answers is not in the document]
Comment (senocak) [reply src/main/resources/tr.yml]: Empty suffix was causing an exception. Removed that. (I'll take a look that later.)

**Review by kingthorin — APPROVED** [review]

**Pull request merged**